src/server/terminal/TerminalResponder.cpp

```diff
@@ -20,5 +20,31 @@ void TerminalResponder::respond(const std::string &str) {
 		if (it != globalPlugins.end())
 			it->second = false;
 	}
+
+	if (str == "load json_stat_api") {
+		plugin::PluginContainer::pluginIterator it = globalPlugins.find("json_stat_api");
+		if (it != globalPlugins.end())
+			it->second = true;
+	}
+	else if (str == "unload json_stat_api") {
+		plugin::PluginContainer::pluginIterator it = globalPlugins.find("json_stat_api");
+		if (it != globalPlugins.end())
+			it->second = false;
+	}
+	else if (str == "exit") {
+		//TODO shutdown
+		(void)str;
+	}
+	else {
+		std::cout	<< "\tcommand options:" 		<< std::endl
+					<< "\t\t- exit" 				<< std::endl
+					<< "\t\t- help" 				<< std::endl << std::endl
+					<< "\tload/unload plugins:" 	<< std::endl
+					<< "\t\t- load page_404_gen"	<< std::endl
+					<< "\t\t- unload page_404_gen"	<< std::endl
+					<< "\t\t- load json_stat_api" 	<< std::endl
+					<< "\t\t- unload json_stat_api" << std::endl;
+	}
+
 	globalLogger.logItem(logger::INFO, "Handled terminal command :)");
 }
```

Comment:
Handle commands differently:
1. split on whitespace (args[])
2. compare args[0] with list of available commands. if not found send usage
3. for load/unload, find args[1] (the plugin), display error if not found and if already loaded/unloaded
4. make sure to send success

Reply:
And for exit, pass a Server class pointer to the arguments of respond(); (which is `this` when you call it)
then just call `server->shutdown();` to exit